src/test/java/org/prebid/server/settings/JdbcApplicationSettingsTest.java

```diff
@@ -661,6 +661,8 @@ private JdbcClient jdbcClient() {
                 new JsonObject()
                         .put("url", JDBC_URL)
                         .put("driver_class", "org.h2.Driver")
-                        .put("max_pool_size", 10)), metrics, clock);
+                        .put("max_pool_size", 10)
+                        .put("provider_class", "io.vertx.ext.jdbc.spi.impl.C3P0DataSourceProvider")),
```

Comment:
replace url on 662 line with jdbcUrl and 665 with hikari type

Reply:
it doesn't need when we use c3p0